lnbits/core/crud.py

```diff
@@ -173,9 +179,10 @@ async def update_user_password(data: UpdateUserPassword) -> Optional[User]:
     pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
 
     await db.execute(
-        "UPDATE accounts SET pass = ? WHERE id = ?",
+        "UPDATE accounts SET pass = ?, updated_at = ? WHERE id = ?",
         (
             pwd_context.hash(data.password),
+            db.timestamp_now,
```

Comment:
does this actually work? `db.timestamp_now` just returns a string and not an actual timestamp

Reply:
take a look again pls :)